vite.config.ts

```diff
@@ -24,23 +40,29 @@ export default defineConfig({
         plugins: [['optimize-clsx', { functionNames: ['getCellClassname'] }]]
       }
     }),
-    !isTest && wyw({ preprocessor: 'none' })
+    wyw({ preprocessor: 'none' })
   ],
   server: {
-    open: true
+    // TODO: open bug
+    open: !isTest
```

Comment:
seems fine now
We could even enable `browser.headless: true`
```suggestion
    open: true
```